src/prpy/planning/tsr.py

```diff
@@ -141,41 +141,46 @@ def PlanToTSR(self, robot, tsrchains, tsr_timeout=2.0,
         valid_solutions = ik_solutions[valid_idxs, :]
         ranked_indices = numpy.argsort(valid_scores)
         ranked_ik_solutions = valid_solutions[ranked_indices, :]
-
-        # Group the IK solutions into sets of the specified size
-        # (plan for each set of IK solutions together).
-        ranked_ik_solution_sets = [
-            ranked_ik_solutions[i:i + chunk_size, :]
-            for i in range(0, ranked_ik_solutions.shape[0], chunk_size)
-        ]
-
-        num_attempts = min(len(ranked_ik_solution_sets), num_attempts)
-        ik_set_list = enumerate(ranked_ik_solution_sets[:num_attempts])
+        ranked_ik_solutions = list(ranked_ik_solutions)
 
         # Configure the robot to use the active manipulator for planning.
+        from openravepy import CollisionOptions, CollisionOptionsStateSaver
+
         p = openravepy.KinBody.SaveParameters
-        with robot.CreateRobotStateSaver(p.ActiveDOF):
+        with robot.CreateRobotStateSaver(p.ActiveDOF),\
+              CollisionOptionsStateSaver(self.env.GetCollisionChecker(), 
+                                         CollisionOptions.ActiveDOFs):
```

Comment:
Unfortunately parens get interpreted as a tuple, and it doesn't work (that was the first thing we tried).  See http://stackoverflow.com/a/31571166.